.github/workflows/handle-external-events.yml

```diff
@@ -126,6 +126,20 @@ jobs:
         env:
           VERSION: ${{ github.event.client_payload.data.version }}
 
+  update-playground-image-on-release:
+    needs: build-release-musl-docker-image
+
+    name: Update https://playground.ponylang.io/
+    runs-on: ubuntu-latest
+    steps:
+      - name: Run remote command
+        uses: appleboy/ssh-action@ffff33f8fe0318345a4f00f0e847325954b2a3ed
```

Comment:
for security reasons, we are using a specific commit hash rather than a branch or tag that could change out from under us.